vexim/siteaddsubmit.php

```diff
@@ -143,25 +139,10 @@
       die;
     }
   } else if ($_POST['type'] == "alias") {
-    $query = "SELECT domain_id FROM domains
```

Comment:
Have you tested how it fails? I haven't, but I suspect not gracefully at all. I'd rather see a separate check and redirect to `site.php?baddestdom=$domain_id` and an error message easy to comprehend instead.